### PR TITLE
KTL-581: Structure

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

File: src/components/Person/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            />
+            <p className={style.info}>
+                <span className={cn('ktl-h4', 'n')}>{name}</span>
+                <span className={cn('ktl-text-2', 'org')}>{company}</span>
+            </p>
+        </div>
+    }}
+/>;
+
+export default Person;
```
Comment (nikpachoo) [inline src/components/Person/index.tsx:41]: I would suggest not to use default import at all. https://notes.webutvikling.org/default-exports/

File: src/pages/structure/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        { name: 'Jan Cichy', company: 'JetBrains' },
+                        { name: 'Alina Dolgikh', company: 'JetBrains' },
+                        { name: 'Hilary Richardson', company: 'Google' },
+                        { name: 'Isabella Fiterman', company: 'Google' },
+                    ]}
+                >
+                    The Trademark committee is responsible for managing the proper use of the Kotlin trademark, which includes its general purpose use as outlined by the Trademark Usage Guidelines, as well as handling specific use-cases that may fall out of the norm. The trademark committee reports directly to the Foundation’s board of directors albeit mostly has autonomy on matters regarding trademarks.
+
+                    <ul className={styles.list}>
+                        <li><a className="layout-link ktl-text-2" href="" target="_blank" rel="noopener noreferrer">Kotlin Brand Usage guidelines</a></li>
```
Comment (nikpachoo) [inline src/pages/structure/index.tsx:77]:
```suggestion
                        <li><a className="layout-link ktl-text-2" href="https://kotlinlang.org/docs/guidelines.html" target="_blank" rel="noopener noreferrer">Kotlin Brand Usage guidelines</a></li>
```

**Review by krutilov — APPROVED** [review]

File: src/pages/structure/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                    hero
+                    name={'Board of Directors'}
+                    members={[
+                        { name: 'Maxim Shafirov', company: 'JetBrains' },
+                        { name: 'Roman Elizarow', company: 'JetBrains' },
+                        { name: 'Grace Kloba', company: 'Google' },
+                        { name: 'Anwar Ghuloum', company: 'Google' },
+                    ]}
+                >
+                    The Foundation is governed by the Board of Directors that makes all decisions by voting. The Board appoints the Lead Language Designer and the members of the Language Committee
```
Comment (nikpachoo) [inline src/pages/structure/index.tsx:25]:
```suggestion
                    The Foundation is governed by the Board of Directors that makes all decisions by voting. The Board appoints the Lead Language Designer and the members of the Language Committee.
```

File: src/components/Structure/structure.module.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.title, .members, .description {
+    /*border-top: 1px solid rgba(39, 40, 44, 0.2);*/
+    padding: 23px 0 30px;
+    margin: 0;
+}
+
+.title::before {
+    background: rgba(39, 40, 44, 0.2);
```
Comment (nikpachoo) [inline src/components/Structure/structure.module.css:33]:
```suggestion
    background: var(--ktl-light-dark-20);
```

**Review by nikpachoo — COMMENTED** [review]
As I see in the DevTools inspector, the Inter font is not connected to the page.
You can import it from RescUI:
```
import '@rescui/typography/lib/font-inter.css'; 
```

**Review by nikpachoo — COMMENTED** [review]
Please double-check the page background color. I guess it should be #F4F4F4 instead of #E5E5E5.

**Pull request merged**